pkgs/servers/kanidm/default.nix

```diff
@@ -24,12 +24,12 @@ rustPlatform.buildRustPackage rec {
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
-    rev = version;
-    hash = "sha256-UavMiHe91UrCZfmG6b+yhdduOY2eKMev9HSjtXq1Tlw=";
+    rev = "d407844e6a7e38f2a44d5aba181ee23054bf25e0";
+    hash = "sha256-YXKMKoqxmdfRJ6wnGEjq4rNph9TA/pW8IVJ4YgibfCo=";
   };
 
   cargoLock = {
-    lockFile = ./Cargo.lock;
+    lockFile = "${src}/Cargo.lock";
```

Comment:
This is triggering IFD which is stricly forbidden

Reply:
You are right m. Thanks you